### DISCUSSION TITLE
speaker diarization custom dataset creation

### DISCUSSION THREAD

**Discussion by arrrrr3186** [pyannote/pyannote-audio#1697] [General]
Hello @hbredin,

I'm currently encountering issues with the performance of a speaker diarization model and I'm considering creating my own dataset to improve its results. Could anyone provide guidance on the steps and tools required to create a custom dataset for this purpose? 

Thank you!

**Comment by arrrrr3186** [comment]
Hello @hbredin,

if you can guide me on this, that would be a great help to me.

Regards,

**Reply by FrenchKrab** [reply]
I assume you have an existing set of audio data and associated annotations that you want to finetune your model on.
Please take a look at the resources in this comment to set it up with pyannote:
https://github.com/pyannote/pyannote-audio/issues/180#issuecomment-2157510117
